bigtable/scheduled-backups/src/main/java/com/example/cloud/bigtable/scheduledbackups/CreateBackupREST.java

```
@@ -0,0 +1,127 @@
+
+        URL url = new URL(BT_ADMIN_BASE_URL + "/projects/" + cbMessage.getProjectId()
+                              + "/instances/" + cbMessage.getInstanceId()
+                              + "/clusters/" + cbMessage.getClusterId()
+                              + "/backups?backupId=" + buildBackupId(cbMessage.getTableId()));
+
+        conn = (HttpURLConnection) url.openConnection();
+        conn.setDoOutput(true);
+        conn.setRequestMethod("POST");
+        conn.setRequestProperty("Content-Type", "application/json");
+
+        String sourceTableField = "projects/" + cbMessage.getProjectId()
+            + "/instances/" + cbMessage.getInstanceId()
+            + "/tables/" + cbMessage.getTableId();
+
+        CreateBackupRequestHTTP request = new CreateBackupRequestHTTP();
+        request.setSourceTable(sourceTableField);
+        request.setExpireTime(buildExpireTime(cbMessage.getExpireHours()));
+
+        String requestBody = mapper.writeValueAsString(request);
+
+        OutputStream os = conn.getOutputStream();
+        os.write(requestBody.getBytes());
+        os.flush();
+
+        if (conn.getResponseCode() != 200) {
+          logger.warning("Create backup of Bigtable table:" + cbMessage.getTableId()
+                             + " failed with response code " + conn.getResponseCode());
+        }
+
+        logger.info(conn.getResponseMessage());
+        BufferedReader br = new BufferedReader(new InputStreamReader((conn.getInputStream())));
+
+        String output;
+        logger.info("Response .... \n");
+        while ((output = br.readLine()) != null) {
+          logger.info(output);
+        }
+      } catch (Exception e) {
+        logger.log(Level.WARNING, "Caught Exception running a BT backup function:"
+            + e.toString(), e);
+      } finally {
+        try {
+          if (conn != null) {
+            conn.disconnect();
+          }
+        } catch (Exception fex) {
```

Same question about catching exceptions here as before. Is swallowing this exception necessary? If it is, please catch a more specific exception type.